### ISSUE TITLE
'High Contrast Black' theme Avatar (TileIcon Background) color

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#998]
**Describe the bug**
TileIcon Background component in the Avatar component has the same text and background-color: white

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/avatar.html
2. Check TileIcon Background component

**Expected behavior**
TileIcon Background component in the Avatar component should have a `color: #000`

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81731122-5fd90800-9443-11ea-9191-d1ec2f37d3c8.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio now 21:1
<img width="375" alt="Screen Shot 2020-10-07 at 6 43 04 PM" src="https://user-images.githubusercontent.com/21978807/95530652-3e009480-0993-11eb-81de-231f44e822df.png">

**Comment by prsdthkr** [comment]
Check with 'High Contrast Dark' theme here:
https://deploy-preview-1752--fundamental-styles.netlify.app/?path=/docs/components-avatar--icon
![image](https://user-images.githubusercontent.com/43764832/95620445-09d5b400-0a25-11eb-9a25-7789363def10.png)
